director/migrations/versions/3f8466b16023_add_users_table.py

```diff
@@ -0,0 +1,40 @@
+"""add users table
```

Comment:
Can you add the uppercase plz ? Just to be consistent with the other migrations :

```
$ director db upgrade
INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
INFO  [alembic.runtime.migration] Will assume transactional DDL.
INFO  [alembic.runtime.migration] Running upgrade  -> 30d6f6636351, Initial migration
INFO  [alembic.runtime.migration] Running upgrade 30d6f6636351 -> 05cf96d6fcae, Add task result
INFO  [alembic.runtime.migration] Running upgrade 05cf96d6fcae -> 3f8466b16023, add users table
```